apertium-crh.crh.lexc

```diff
@@ -8,65 +9,82 @@
 !!!                                                                         !!!
 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
 
+! See List of symbols http://wiki.apertium.org/wiki/List_of_symbols
+
+! See http://wiki.apertium.org/wiki/Crimean_Tatar and http://wiki.apertium.org/wiki/Category:Turkic_languages
+
 Multichar_Symbols 
 
-%<n%>             ! Noun                        !
-%<np%>            ! Proper noun                 !
-%<adj%>           ! Adjective                   !
-%<prn%>           ! Pronoun                     !
-%<post%>          ! Postposition                !
-%<adv%>           ! Adverb                      !
-%<v%>             ! Verb                        !
-%<cnjcoo%>        ! Coordinating conjunction    !
-%<cnjsub%>        ! "subordinating conjunction" (ki)
-%<cnjadv%>        ! 
+%<n%>             ! Noun                        
+%<np%>            ! Proper noun                 
+%<adj%>           ! Adjective                   
+%<prn%>           ! Pronoun                     
+%<post%>          ! Postposition                
+%<adv%>           ! Adverb                      
+%<v%>             ! Verb                        
+%<cnjcoo%>        ! Coordinating conjunction    
+%<cnjsub%>        ! Sub-ordinating conjunction
+%<cnjadv%>        ! Adverbial conjunction 
 %<qst%>           ! Question word
-%<sym%>
+%<sym%>           ! Symbol
 %<cop%>           ! Copula
-%<postadv%>       ! 
-%<det%>           ! 
-%<num%>           ! 
+%<postadv%>       ! Post Adverb
+%<det%>           ! Determiner
+%<num%>           ! Numeral
 %<vaux%>          ! Auxiliary verb
 %<abbr%>          ! Abbreviation
-%<emph%> 
-%<ij%>
-%<nom%>           ! Nominative                  !
-%<gen%>           !
-%<dat%>           !
-%<acc%>           !
-%<loc%>           ! Locative                    ! 
-%<abl%>           ! Ablative                    !
-%<ins%>           !
-%<sim%>
-
-%<iv%>            !
-%<tv%>            !
-
-%<nec%>
-%<imp%>
-%<aor%>
-%<gno%>
-%<fut%>
-%<fut2%>          ! -EcAG
```

Comment:
Why were these removed?

Reply:
They arent removed they all are present in .lexc file but they are scattered unlike above but they are present in it pls see

Reply:
What line number?

Reply:
aor-65
gno-76
fut-77
fut2-78